MdeModulePkg/Bus/Pci/SdMmcPciHcDxe/SdMmcPciHci.c

```diff
@@ -801,39 +801,37 @@ SdMmcHcStartSdClock (
 
   Refer to SD Host Controller Simplified spec 3.0 Section 3.2.1 for details.
 
-  @param[in] Private         A pointer to the SD_MMC_HC_PRIVATE_DATA instance.
-  @param[in] Slot            The slot number of the SD card to send the command to.
-  @param[in] BusTiming       BusTiming at which the frequency change is done.
-  @param[in] FirstTimeSetup  Flag to indicate whether the clock is being setup for the first time.
-  @param[in] ClockFreq       The max clock frequency to be set. The unit is KHz.
+  @param[in] PciIo          The PCI IO protocol instance.

```

Comment:
@aimanrosli23 
For the changes in file SdMmcPciHci.c, please make sure that you do not revert the changes made by the below commits:
SHA-1: 49accdedf956f175041040e677163b7cbb746283
* MdeModulePkg/SdMmcPciHcDxe: Hook SwitchClockFreq after SD clock start

SHA-1: c67617f3c677c342efde780e229f841f4e0f6c7e
* MdeModulePkg/SdMmcPciHcDxe: Add retries for sync commands

SHA-1: 6d387610855ee27b49cb720de29ed809ea59dadc
* MdeModulePkg/SdMmcPciHcDxe: Read response on command completion

SHA-1: 63fd7f389874081ea37452b1f79ac013d52755e3
* MdeModulePkg/SdMmcPciHcDxe: Do not map memory for non DMA transfer